packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -761,6 +793,72 @@ export class Combobox
     this.updateActiveItemIndex(targetIndex);
   }
 
+  private hideChip(chipEl: HTMLCalciteChipElement): void {
+    chipEl.classList.remove(CSS.chipVisible);
+    chipEl.classList.add(CSS.chipInvisible);
+  }
+
+  private showChip(chipEl: HTMLCalciteChipElement): void {
+    chipEl.classList.remove(CSS.chipInvisible);
+    chipEl.classList.add(CSS.chipVisible);
+  }
+
+  private refreshDisplayMode = () => {
+    if (isSingleLike(this.selectionMode)) {
+      return;
+    }
+    if (!this.textInput) {
+      return;
+    }
+    if (this.displayMode === "fit-to-line") {
+      const chipEls = Array.from(this.el.shadowRoot.querySelectorAll("calcite-chip")).filter(
+        (chipEl) => chipEl !== this.selectedIndicatorChipEl
+      );
+      const { fontSize, fontFamily } = getComputedStyle(this.textInput);
+      const inputContainerElWidth = getElementWidth(this.inputContainerEl);
+      const inputContainerElGap = parseInt(
+        getComputedStyle(this.inputContainerEl).gap.replace("px", "")
```

Comment:
Might be able to reference the `gap` value from a design token here, need to look into that.